cmd/clusterctl/examples/openstack/ubuntu/machines.yaml.template

```diff
@@ -19,6 +19,7 @@ items:
         floatingIP: 129.114.111.153
         securityGroups:
         - default
+        userDataSecret: master-user-data
```

Comment:
Might change if you decide to change type of userDataSecret to `SecretReference`. See other comment.